core/services/workflows/metering/metering.go

```diff
@@ -572,11 +574,18 @@ func (r *Report) getMaxSpendForInvocation(
 }
 
 func (r *Report) switchToMeteringMode(err error) {
+	// add to the reported errors to indicate all errors encountered during metering
+	r.meteringModeErr = errors.Join(r.meteringModeErr, err)
+
+	if r.meteringMode {
+		return
+	}
+
+	// only log a single metering mode switch error. this error should indicate the first metering related error
+	// encountered
 	r.lggr.Errorf("switching to metering mode: %s", err)
```

Comment:
We were logging more than once again after moving the billing rate card calls to the report initialization step.